### DISCUSSION TITLE
Cyberduck Read Timed Out Unsuccessful Downloads

### DISCUSSION THREAD

**Discussion by apl000** [iterate-ch/cyberduck#15402] [Q&A]
**Describe the bug**
Uploading files to Backblaze B2 results to unsuccessful uploads with the error message that says "read timed out"

**To Reproduce**
1. Opened Backblaze B2 with Cyberduck.
2. Chose a bucket.
3. Uploaded a file
4. Unable to successfully upload, see error message.

**Expected behavior**
Unsuccessful uploading. Transfers show that the file upload is complete but the file is nowhere to be found on the chosen bucket and folder.

**Screenshots**
![image](https://github.com/iterate-ch/cyberduck/assets/151890805/87b43bf1-fbb7-490a-b05a-fa5b5c279e20)
![image](https://github.com/iterate-ch/cyberduck/assets/151890805/38fc1e04-a29e-4e52-a9e5-c9d39323e956)

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro
 - Version 22H2

**Log Files**
[cyberduck.log](https://github.com/iterate-ch/cyberduck/files/13469380/cyberduck.log)

**Additional context**
Was able to successfully utilize uploading using Cyberduck for a few days. Then it stopped working showing the error message shown above.

**Comment by dkocher** [comment]
Most probably finishing the large file upload fails because of the B2 servers not responding in time. Try to increase _Preferences → Connection → Timeouts → Repeat failed networking tasks_ and _Preferences → Connection → Timeouts → Timeout for opening connections_. If that does not resolve the problem reach out to Backblaze B2 customer support.

**Reply by apl000** [reply]
I have tried changing the preferences. Still no success. I already reached out to Backblaze B2 customer support about the matter. Even Backblaze B2 told me to reach out to Cyberduck customer support lol.

**Reply by dkocher** [reply]
It is a bug that the transfer is shown as complete when all parts have been successfully submitted but finishing the large upload fails with B2 with a timeout. However the bad performance of the B2 API is a well known issue and should be resolved by their team.

**Reply by apl000** [reply]
I tried to upload again just now using Cyberduck. I tried uploading a file that is 2.7 MB. To my surprise, it was uploaded successfully. But then again after trying to upload another file, it showed the same reoccurring error over again.
